### ISSUE TITLE
ROI selector locks color map

### ISSUE THREAD

**Issue by dani-lbnl** [InsightSoftwareConsortium/itkwidgets#229]
Starting 0.23.0, when selecting a ROI, there seems to be a default colormap that kicks in and get locked into - in other words, if using ROI subregion, there is no variety of colormap that can be used, just a blue-pink colormap, which is less helpful than for example, viridis.
Link to Example:
https://www.dropbox.com/s/5huzk9svuvykzih/Screenshot%202019-11-20%2013.39.22.png?dl=0

**Comment by thewtex** [comment]
@dani-lbnl thanks for the report :+1: 

This blue-pink colormap is the vtk.js default: our setting is not being applied.

To help understand the behavior --

1. Is the default viridis colormap applied as expected?
2. If a `cmap` keyword argument is passed to the `view` function to select a colormap, is its value applied?
3. If a different colormap is selected via the widget interactive user interface, is it applied?

**Comment by rpfeynman** [comment]
Hi! I have the same problem, so I think I can answer those questions (at least for my case).

1. Yes it is, however I can't change the dynamic range and pretty much every colormap-related setting in the UI don't work
2. Yes, it is applied until one clicks on the ROI button and moves one of the corners. Then the colormaps reverts to the Blue-Pink one and sticks to that forever.
3. No, there's no way to change it back rather than running the cell again.

Also, it looks like all other settings are reset, for example the volume moves to the center of the cell output
![view_colormap_issue](https://user-images.githubusercontent.com/3237202/70820130-d1972200-1dcf-11ea-9261-7bf58117c286.gif)

**Comment by rpfeynman** [comment]
Actually, while playing with this I just discovered that this issue happens only one of the entries of `size_limit_3d` is smaller than the corresponding volume dimension.

So an easy workaround for this issue is to call the `view` function as 
```python
view(image, size_limit_3d=image.GetLargestPossibleRegion().GetSize())
```

**Comment by thewtex** [comment]
@dani-lbnl @rpfeynman thanks for the report and description! I was able to reproduce this, and it is addressed in #234 , to be released in itkwidgets version 0.24.0.

**Comment by thewtex** [comment]
> @dani-lbnl @rpfeynman thanks for the report and description! I was able to reproduce this, and it is addressed in #234 , to be released in itkwidgets version 0.24.0.

Correction: this is actually in 0.24.1.
